base/libc.jl

```diff
@@ -277,7 +277,7 @@ if is_windows()
         const FORMAT_MESSAGE_FROM_SYSTEM = UInt32(0x1000)
         const FORMAT_MESSAGE_IGNORE_INSERTS = UInt32(0x200)
         const FORMAT_MESSAGE_MAX_WIDTH_MASK = UInt32(0xFF)
-        lpMsgBuf = Array(Ptr{UInt16})
+        lpMsgBuf = Array{Ptr{UInt16},0}()
```

Comment:
Not necessary to change in this PR since this is the exact equivalent of what was there, but shouldn't this be updated to use a `Ref{Ptr{UInt16}}` instead of a zero-dimensional array?